src/main/java/edu/uc/inventorymanager/dto/Item.java

```diff
@@ -38,8 +38,12 @@
 
     @Override
     public boolean equals(Object o) {
-        if (this == o) return true;
-        if (o == null || Hibernate.getClass(this) != Hibernate.getClass(o)) return false;
+        if (this == o) {
+            return true;
+        }
+        if (o == null || Hibernate.getClass(this) != Hibernate.getClass(o)) {
+            return false;
+        }
```

Comment:
Opening and closing curly braces should not be omitted from if statements as they make the if statement more readable. In my opinion, it also can make it easier to change the if statements should there be a need for more than just a simple return in the statement. Source: https://rhamedy.medium.com/a-short-summary-of-java-coding-best-practices-31283d0167d3